apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/service/OnboardingService.java

```diff
@@ -100,9 +99,10 @@ public void createContract(OnboardingWorkflow onboardingWorkflow) {
         contractService.createContractPDF(onboardingWorkflow.getContractTemplatePath(product), onboarding, manager, delegates, product.getTitle(), onboardingWorkflow.getPdfFormatFilename());
     }
 
+    // TODO
     public void loadContract(Onboarding onboarding) {
         Product product = productService.getProductIsValid(onboarding.getProductId());
-        contractService.loadContractPDF(product.getContractTemplatePath(), onboarding.getId(), product.getTitle());
+        contractService.loadContractPDF(product.getInstitutionContractTemplate(onboarding.getInstitution().getInstitutionType().name()).getContractTemplatePath(), onboarding.getId(), product.getTitle());
```

Comment:
could you check if onboarding.getInstitution() is null?

Reply:
Fixed!